x-pack/plugins/lens/server/plugin.tsx

```diff
@@ -21,33 +17,41 @@ import {
 } from './usage';
 
 export interface PluginSetupContract {
-  savedObjects: SavedObjectsLegacyService;
   usageCollection: UsageCollectionSetup;
-  config: KibanaConfig;
-  server: Server;
   taskManager: TaskManagerSetupContract;
 }
 
 export interface PluginStartContract {
-  server: Server;
   taskManager: TaskManagerStartContract;
 }
 
 const taskManagerStartContract$ = new Subject<TaskManagerStartContract>();
 
```

Comment:
If you now have TaskManager as a required dependancy then you should no longer need this Subject as the NP should ensure you have the StartContract available to you.

Reply:
This is a subject for a different reason - as we have to register the usage collector in the setup phase but need the start contract of the task manager to do so, we have to somehow provide it async. I'm not super happy with this solution, if you have a better one I'm glad to improve.